resources/asciidoctor/spec/edit_me_spec.rb

```diff
@@ -81,7 +82,8 @@ def spec_dir_link(file)
       end
     end
 
-    shared_examples 'standard document part' do |type, title_start = '<title>', title_end = '</title>'|
+    shared_examples 'standard document part' \
+        do |type, title_start = '<title>', title_end = '</title>'|
```

Comment:
This one kind of bothers me but it is the way that make rubocop happy.

Reply:
looks a bit strange to me. Maybe assign `title_start` and title_end` in the block below.